src/scrabble.js

```diff
@@ -1,5 +1,126 @@
-function scrabble() {
+const isStringZero = (str) => {
+  if (str.length === 0) {
+    str += '0'
+  }
+  return str
+}
+
+const stringToArray = (word) => {
+  const letters = []
+  word
+    .toLowerCase()
+    .split('')
+    .forEach((element) => {
+      letters.push(element)
+    })
+  return letters
+}
+
+const checkWordMulti = (array) => {
+  let wordMulti = 1
+  let wordMultis = 0
+  const datasheet = require('./datasheet.js').datasheet
+  if (Object.getOwnPropertyNames(datasheet).includes(array[0])) {
+    return [array, wordMulti]
+  } else if (
+    (array[0] === '[' && array[2] === ']') ||
+    (array[0] === '{' && array[2] === '}')
+  ) {
+    return [array, wordMulti]
+  } else {
+    for (
+      let i = 0;
+      !Object.getOwnPropertyNames(datasheet).includes(array[i]);
+      i++
+    ) {
+      if (array[i] === '[' && array[array.length - (1 + i)] === ']') {
```

Comment:
what is the condition in the if statement on line 36 telling us?

Reply:
it checks if the bracket is around a character. so if the input would be `[, a, ]` it would check if index 0 and index 2 are a pair of brackets indicating that the letter in between should have a multiplier applied to it.